dev/podman/kind-with-registry-podman.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/sh
+set -o errexit
+
```

Comment:
Any need/desire to add a check here to make sure `podman` is actually installed? (e.g. test `command -v podman`)

Reply:
No need IMO since the readme mentions Podman as a prerequisite.